<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourcePage</class>
 <widget class="QWidget" name="ResourcePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>‚ÄúSegoe UI‚Äù</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #345;
    font: 15px ‚ÄúSegoe UI‚Äù;
	font:bold;
}</string>
   </property>
   <property name="text">
    <string>ü§ù Resource Sharing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>47</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>‚ÄúSegoe UI‚Äù</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #345;
    font: 13px ‚ÄúSegoe UI‚Äù;
	font:bold;
}</string>
   </property>
   <property name="text">
    <string>Reource Item:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editItem">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>50</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>‚ÄúSegoe UI‚Äù</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #345;
    font: 13px ‚ÄúSegoe UI‚Äù;
	font:bold;
}</string>
   </property>
   <property name="text">
    <string>Availability:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editAvailability">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>90</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>126</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color: #5a9;
    border: none;
    border-radius: 6px;
    font: 14px ‚ÄúSegoe UI‚Äù, bold;
    color: white;
}
QPushButton:hover {
    background-color: #6ba;
}
QPushButton:pressed {
    background-color: #498;
}</string>
   </property>
   <property name="text">
    <string>‚úÖ Submit Resource</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>‚ÄúSegoe UI‚Äù</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #345;
    font: 13px ‚ÄúSegoe UI‚Äù;
	font:bold;
}</string>
   </property>
   <property name="text">
    <string>‚ÑπÔ∏èStatus:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>‚ÄúSegoe UI‚Äù</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #345;
    font: 13px ‚ÄúSegoe UI‚Äù;
	font:bold;
}</string>
   </property>
   <property name="text">
    <string>Submit Resource to Check Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>200</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>‚ÄúSegoe UI‚Äù</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #345;
    font: 13px ‚ÄúSegoe UI‚Äù;
	font:bold;
}</string>
   </property>
   <property name="text">
    <string>üì¶ Availible Resources:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblResources">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>471</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: #ffffff;
    border: 1px solid #ccc;
    border-radius: 8px;
    gridline-color: #e0e0e0;
    selection-background-color: #a8d5e2;
    selection-color: #003344;
    font: 13px &quot;Segoe UI&quot;, sans-serif;
}

/* Horizontal header */
QHeaderView::section {
    background-color: #f0f4f8;
    color: #333;
    padding: 8px;
    border: none;
    border-bottom: 1px solid #ccc;
    font-weight: bold;
}

/* Vertical header (hide if desired) */
QTableView QHeaderView::verticalHeader {
    background: transparent;
    border: none;
}

/* Rows */
QTableWidget::item {
   /* padding: 6px;*/
}

/* Alternate row coloring */
QTableWidge {
    alternate-background-color: #f9fcff;
}

/* Hover effect */
QTableWidget::item:hover {
    background-color: #d0eef7;
}

/* Selected row */
QTableWidget::item:selected {
    background-color: #a8d5e2;
    color: #003344;
}

/* Remove focus rectangle */
QTableWidget#tblResources::item:focus {
    outline: none;
}</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Item</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Owner</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Availability</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Actions</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>UPS</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>Ali</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>9 am</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>Generator</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>Ahmed</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>10 pm</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>Water Bottles</string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>Sara</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>12 pm</string>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string>Vaccum</string>
    </property>
   </item>
   <item row="3" column="1">
    <property name="text">
     <string>Hasan</string>
    </property>
   </item>
   <item row="3" column="2">
    <property name="text">
     <string>8 am</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
